src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useEffect, useRef, useState } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { updateTodo } from '../../api/todos';
+import { Errors } from '../../types/Errors';
+
+type Props = {
+  todo: Todo;
+  onDeleteTodo?: (todoId: number) => void;
+  onUpdateTodos?: (newTodo: Todo) => void;
+  loadingTodoIds?: number[];
+  onError?: (error: Errors) => void;
+};
+
+export const TodoInfo: React.FC<Props> = ({
+  todo,
+  onDeleteTodo = () => {},
+  onError = () => {},
+  onUpdateTodos = () => {},
+  loadingTodoIds,
+}) => {
+  const [editTitle, setEditTitle] = useState(todo.title);
```

Comment:
Use[ destruction assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) to a todo object